config/runtimes/vllm-multinode-template.yaml

```diff
@@ -0,0 +1,256 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  labels:
+    opendatahub.io/dashboard: "true"
+    opendatahub.io/ootb: "true"
+  annotations:
+    description: vLLM is a high-throughput and memory-efficient inference and serving engine for LLMs
+    openshift.io/display-name: vLLM ServingRuntime for KServe
+    openshift.io/provider-display-name: Red Hat, Inc.
+    tags: rhods,rhoai,kserve,servingruntime
+    template.openshift.io/documentation-url: https://github.com/opendatahub-io/vllm
+    template.openshift.io/long-description: This template defines resources needed to deploy vLLM ServingRuntime with KServe in Red Hat OpenShift AI
+    opendatahub.io/modelServingSupport: '["single"]'
+    opendatahub.io/apiProtocol: "REST"
+  name: vllm-multinode-runtime-template
+objects:
+  - apiVersion: serving.kserve.io/v1alpha1
+    kind: ServingRuntime
+    metadata:
+      name: vllm-multinode-runtime
+      annotations:
+        openshift.io/display-name: vLLM ServingRuntime for KServe
+        opendatahub.io/recommended-accelerators: '["nvidia.com/gpu"]'
+      labels:
+        opendatahub.io/dashboard: "true"
+    spec:
+      annotations:
+        prometheus.io/port: "8080"
+        prometheus.io/path: "/metrics"
+      multiModel: false
+      supportedModelFormats:
+        - autoSelect: true
+          name: vLLM
+          priority: 2
+      containers:
+        - name: kserve-container
+          image: $(vllm-image)
+          command: ["bash", "-c"]
+          args:
+            - |
+              ray start --head --disable-usage-stats --include-dashboard false 
+              # wait for other node to join
+              until [[ $(ray status | grep -c node_) -eq ${PIPELINE_PARALLEL_SIZE} ]]; do
+                echo "Waiting..."
+                sleep 1
+              done
+              ray status
+
+              export SERVED_MODEL_NAME=${MODEL_NAME}
+              export MODEL_NAME=${MODEL_DIR}
+
+              exec python3 -m vllm_tgis_adapter
```

Comment:
@vaibhavjainwiz @Xaenalt I used `vllm_tgis_adapter` based on IBM PoC. Runtime team is ok to use this?

Reply:
It'd be better to have `python -m vllm.entrypoints.openai.api_server` just like we have in the vLLM standalone template

Reply:
ok I will use this